### PR TITLE
openbsd_pkg: Fix when failed, PEP8-compliant

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This includes:
- PEP8 compliancy
- A fix to ensure the module fails when it failed for a package
- Various cosmetic changes to documentation
- Make `state: present` the default (and not required)

Related to PR #25950

##### ISSUE TYPE

 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME

openbsd_pkg

##### ANSIBLE VERSION

v2.4

### CODE REVIEW

**Comment by eest** [comment]
Thank you for looking at this, I wasn't aware the module was on a legacy list :).

There is currently another PR in the pipe, #26016. Do you think this can wait until that one has been merged?

Also, just out of curiosity, what is the reason for adding an "**result" argument to module.fail_json? Should it be done for all invocations of module.fail_json or only for this final one?

**Comment by eest** [comment]
@dagwieers: You think you can rebase this against the current version of openbsd_pkg? I can then test it for you.

**Comment by dagwieers** [comment]
@eest Rebased, and various other changes.

**Comment by eest** [comment]
I have now tested the patch successfully. Thank you for the review and resulting fixes :).

shipit

**Comment by eest** [comment]
@bcoca: Maby you can do the final review for this?

File: lib/ansible/modules/packaging/os/openbsd_pkg.py
Comment (bcoca) [inline lib/ansible/modules/packaging/os/openbsd_pkg.py:600]: isn't this redundant with lines above?

**Review by bcoca — CHANGES_REQUESTED** [review]

**Review by bcoca — APPROVED** [review]

**Comment by bcoca** [comment]
rebuild_merge

File: lib/ansible/modules/packaging/os/openbsd_pkg.py
Comment (dagwieers) [reply lib/ansible/modules/packaging/os/openbsd_pkg.py:600]: Hmm, must have been a merge issue when rebasing.

Comment (bcoca) [reply lib/ansible/modules/packaging/os/openbsd_pkg.py:600]: nvmd, i misread

Comment (dagwieers) [reply lib/ansible/modules/packaging/os/openbsd_pkg.py:600]: Haha, and you influenced my reading with that. Mind-control !

**Pull request merged**
